### DISCUSSION TITLE
Add "airbyte" prefix to OTEL metrics name

### DISCUSSION THREAD

**Discussion by MrXander** [airbytehq/airbyte#46572] [Platform Features or Ideas]
Hello everyone,

We have a separate prometheus instance that collects OTEL metrics from a set of services, so that it's hard to find airbyte metrics among others as they don't have any "airbyte" in theirs name. 
Can you please add a prefix "airbyte" or any other to be able to distinguish airbyte otel metrics from others?

Also I couldn't find a full list of OTEL metrics, so that I had to run airbyte, otel collector, grafana locally and do some testing to force airbyte to send metrics. Is there any way to get a full list of all possible metrics?

Thank you

**Comment by mateocolina** [comment]
Would be interested in this feature as well.

[a comment by an automated account is not shown]
